e2e/bzlmod/BUILD.bazel

```diff
@@ -1,10 +1,11 @@
 load("@aspect_rules_js//js:defs.bzl", "js_test")
 load("@npm//:defs.bzl", "npm_link_all_packages")
-load("@npm_meaning-of-life__links//:defs.bzl", npm_link_meaning_of_life = "npm_link_package")
+load("@npm_meaning-of-life__links//:defs.bzl", npm_link_meaning_of_life = "npm_link_imported_package")
 
 npm_link_all_packages(name = "node_modules")
 
-npm_link_meaning_of_life(name = "node_modules/meaning-of-life")
+# buildifier: disable=duplicated-name
```

Comment:
yeah... this seems undesirable, I think we need to fix it.

could it be


```
load("@npm_meaning-of-life__links//:defs.bzl", meaning_of_life = "npm_link_imported_package")
npm_link_all_packages(
    name = "node_modules",
    imported_links = [
        meaning_of_life,
    ]
)
```
??

Reply:
Yea, it's not great but OTOH its only a problem with manual npm_imports.

Users could also do,
```
npm_link_all_packages(name = "node_modules")
npm_link_meaning_of_life()
```
since `node_modules` is default anyway.

But I also like your suggestion.

Reply:
I think your way reads better even for uncommon cases; and it has the added benefit of being able to include the manual npm_imports that are linked direct in a given package to the `:node_modules` catch-all target for that package

Reply:
I'll try it out

Reply:
fixed up

Reply:
I'm not clear what the outcome was, but leaving out `name` from macros makes buildifier mad, also I use this "query" a bunch, so this would yield missing results:

```
buildozer "print name" //...:all
```

Reply:
outcome is to have one target with a `name` field, so all good there